source/backend/api/Areas/Projects/Controllers/ProjectController.cs

```diff
@@ -107,6 +113,29 @@ public async Task<IActionResult> AddProject(ProjectModel projectModel)
             return Ok(newProject);
         }
 
+        /// <summary>
+        /// Updates the project.
+        /// </summary>
+        /// <returns></returns>
+        [HttpPut("{id:long}")]
+        [HasPermission(Permissions.ProjectEdit)]
+        [Produces("application/json")]
+        [ProducesResponseType(typeof(ProjectModel), 200)]
+        [SwaggerOperation(Tags = new[] { "project" })]
+        public IActionResult UpdateProject([FromRoute]long id, [FromBody] ProjectModel model)
+        {
+            _logger.LogInformation(
+                "Request received by Controller: {Controller}, Action: {ControllerAction}, User: {User}, DateTime: {DateTime}",
+                nameof(ProjectController),
+                nameof(UpdateProject),
+                User.GetUsername(),
+                DateTime.Now);
+
+            var updatedProject = _projectService.Update(id, _mapper.Map<Dal.Entities.PimsProject>(model));
+
+            return Ok(updatedProject);
```

Comment:
please use JsonResult, as it is more consistent with our existing patterns (or provide a reason why this is a better alternative).

Reply:
This does not look addressed

Reply:
IMHO: .Net already returns JSON format by default. Returning a new instance of a JsonResult class seems redundant and not usefull. 
I would rather focus on returning the proper response than type/format and let the API configured options to handle/negotiate the response format/type.